### ISSUE TITLE
My favourite list was cleared completely.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [EionRobb/okcupid-pidgin#9]
```
What steps will reproduce the problem?
No idea what I did wrong..

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
I have Windows Vista Premium, just downloaded the latest version of this.

Please provide any additional information below.


```

Original issue reported on code.google.com by `henkvand...@gmail.com` on 14 Nov 2010 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
Did you rename any groups or delete any users from your list?
```

Original comment by `eionrobb` on 14 Nov 2010 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
Yes I did rename the group.
```

Original comment by `henkvand...@gmail.com` on 15 Nov 2010 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
Hmm.. ok, then that might have caused it.  I can add in a workaround, but since 
the buddies aren't in the favourites list any more, you might not be able to 
get them back :(

The more detailed explanation is that when renaming a group, Pidgin normally 
adds all the users again, with the new group name and then removes all the 
users from the old group (by deleting them, thus removing them from your 
favourites).

I can add in a workaround that may fix this problem.  Sorry about that.
```

Original comment by `eionrobb` on 15 Nov 2010 at 8:22
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Ah that's alright (had too many people in that list anyway) but I can imagine 
some people not being that happy about it;) thanks for looking at it!
```

Original comment by `henkvand...@gmail.com` on 15 Nov 2010 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
This bug is fixed in v2.05
```

Original comment by `eionrobb` on 23 Mar 2011 at 11:11
- Changed state: **Fixed**
